packages/mdc-theme/README.md

```diff
@@ -110,6 +110,38 @@ Property Name | Description
 `on-secondary` | A text/iconography color that is usable on top of secondary color
 `on-surface` | A text/iconography color that is usable on top of surface color
 
+#### `mdc-theme-prop` with CSS Custom Properties
+
+> **Note** The Sass map `$style` argument is intended *only* for use with color mixins.
+
+The `mdc-theme-prop` mixin also accepts a Sass map for the `$style` argument. The map must contain the following fields:
+
+Fields | Description
+--- | ---
+`varname` | The name of a CSS custom property
+`fallback` | A fallback value for the CSS custom property
+
+For example, the following Sass...
+
+```
+.foo {
+  @include mdc-theme-prop(color, (
```

Comment:
If I understand this correctly, this is equivalent to applying same mixin with CSS custom property and regular CSS value?

```
mdc-button-fill-color(purple);
mdc-button-fill-color(var(--my-custom-button, purple));
```

Reply:
The output CSS is pretty different. Calling both mixins includes the full selector twice. Below is an approximation of the output CSS:

```
.mdc-button {
  background-color: purple;
}
.mdc-button {
  background-color: var(--my-custom-button, purple);
}
```